main.cpp

```diff
@@ -0,0 +1,77 @@
+/*  
+	main.cpp by Subhed Chavan, Jayesh Sathe, Gopal Raigoni @ Chico State
+    Github @subhed @Jayesh-sathe @gopalsingh112
+    CSCI-630 
+*/
+
+#include <iostream>
+#include "Game_Type.h"
+#include "Human_Player.h"
+#include "Computer_Player.h"
+#include "Records.h"
+
+using namespace std;
+
+Game_Type select_game_type()
+{
+    cout << "1. Press 1 for Single Player" << endl
+         << "2. Press 2 for Dual Player" << endl;
+    int G_Type;
+    cin >> G_Type;
+
+    switch (G_Type)
+    {
+    case 1:
+        GT = Game_Type::single;
+        return GT;
+        break;
+
+    case 2:
+        GT = Game_Type::dual;
+        return GT;
+        break;
+
+    default:
+        break;
+    }
+}
+
+void show_records()
+{
+    Records display = Records::instance();
+    display.show_records();
+}
+
+void show_menu()
+{
+
+    Game GM;
+
+    cout << "1. Press 1 to play the game" << endl
+         << "2. Press 2 to see records" << endl;
+
+    int G_Menu;
+    cin >> G_Menu;
+
+    switch (G_Menu)
+    {
+    case 1:
+        Game_Type GT = select_game_type();
+        GM(GT);
+        show_records();
+        break;
+
+    case 2:
+        show_records();
+        break;
+
+    default:
+        break;
```

Comment:
3 to quit, anything else, print an error message and repeat menu.

Reply:
Feature added to the program.